src/ngx_fetch.cc

```diff
@@ -735,31 +735,34 @@ void NgxFetch::ConnectionWriteHandler(ngx_event_t* wev) {
     if (n >= 0) {
       out->pos += n;
     } else if (n == NGX_AGAIN) {
-      if (ngx_handle_write_event(c->write, 0) != NGX_OK) {
-        fetch->CallbackDone(false);
-      }
-      // Timer set in Init() is still in effect.
-      return;
+      break;
     } else {
-      c->error = 1;
-      fetch->CallbackDone(false);
-      return;
+      ok = false;
```

Comment:
@jeffkaufman Might be, currently we'll log debug messages at a higher level NgxFetch::CallbackDone
I also *think* that nginx will log something when we end up here, though it might be hardish to associate that message with this NgxFetch instance. So, should I just add a warning message?